web/endpoint.go

```diff
@@ -1,3 +1,4 @@
+// Package web provides managing HTTP protocols.
 // Configurate endpoints to interact with other tools of the project.
-
+// Realise CRUD threw filters.
```

Comment:
I'm not really sure what does it mean.

Reply:
Well, now I see that it wasn`t logical for me even to write that comment, cause from the structure of the program it`is clear that there expected such methods. So I deleted it :)